### PR TITLE
Corrected publication.title not showing on light mode

### PR DESCRIPTION
**Pull request by victornwakpa**
When using the light mode, the "publication.title" does not show because there is no light mode color assigned to it.

Fixed that by giving the default color: text-slate-900, then on dark mode: text-white

image

### CODE REVIEW

**Comment by panda-sandeep** [comment]
Thanks for the PR! We’ll take a look tomorrow. :)

File: README.md [hunk 1]
```diff
@@ -38,7 +38,9 @@ The recommended approach is deploying to Vercel. If you don't have an account al
 - Fork this repo
 - Create a new project on Vercel and connect this repo
 - It's a monorepo. So, choose either `packages/blog-starter-kit/themes/enterprise` or `packages/blog-starter-kit/themes/personal` as the root directory while importing on Vercel.
-  ![selecting the directory to deploy a monorepo](https://cdn.hashnode.com/res/hashnode/image/upload/v1695083263935/T5bByLxZT.png?w=500&h=800&auto=format)
+
+  ![selecting the directory to deploy a monorepo](https://github.com/victornwakpa/starter-kit/assets/61851642/f6d25a80-065a-48a2-9501-4047de544eb5)
```
Comment (fazlerocks) [inline README.md:42]: Hey @victornwakpa, could you upload this image to Hashnode's CDN using our uploader? https://hashnode.com/uploader

File: packages/blog-starter-kit/themes/enterprise/components/footer.tsx [hunk 2]
```diff
@@ -20,7 +20,7 @@ export const Footer = () => {
 						</Link>
 					</div>
 				) : (
-					<p className="mb-20 text-center text-xl font-semibold text-white md:text-4xl">
+					<p className="mb-20 text-center text-xl font-semibold text-slate-900 dark:text-white md:text-4xl">
```
Comment (fazlerocks) [inline packages/blog-starter-kit/themes/enterprise/components/footer.tsx:23]: Could you please change this to `dark:text-slate-50`?

File: README.md (hunk 1 above)
Comment (victornwakpa) [reply README.md:42]: okay

File: packages/blog-starter-kit/themes/enterprise/components/footer.tsx (hunk 2 above)
Comment (victornwakpa) [reply packages/blog-starter-kit/themes/enterprise/components/footer.tsx:23]: Alright

File: README.md (hunk 1 above)
Comment (victornwakpa) [reply README.md]: I have applied the correction

File: packages/blog-starter-kit/themes/enterprise/components/footer.tsx (hunk 2 above)
Comment (victornwakpa) [reply packages/blog-starter-kit/themes/enterprise/components/footer.tsx]: I have applied the correction you made

> Could you please change this to `dark:text-slate-50`?

**Pull request merged**